scss/_settings_colors.scss

```diff
@@ -12,21 +13,20 @@ $color-mid: #999 !default;
 $color-mid-dark: #666 !default;
 $color-dark: #111 !default;
 $color-x-dark: #000 !default;
-
-$color-negative: #c7162b !default;
-$color-caution: #f99b11 !default;
-$color-positive: #0e8420 !default;
-$color-information: #335280 !default;
+$color-input-shadow: rgba($color-x-dark, 0.12) !default;
 
 $color-label-validated: #006b75;
 
-$color-input-shadow: rgba($color-x-dark, 0.12) !default;
-
-$color-accent: $color-brand !default;
-$color-accent-background: $color-accent !default;
+$color-negative: #c7162b !default;
+$color-caution: #f99b11 !default;
+$color-positive: hsl(129, 81%, 29%) !default;
```

Comment:
Why switch to use hsl here?

Reply:
shouldn't be, I was just trying to check the hue in vscode - reverted.